app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -67,20 +76,32 @@ class Config {
     final Uint8List cacheValue = await _cache.getOrCreate(
       configCacheName,
       'flutterBranches',
-      createFn: () => getBranches(Providers.freshHttpClient, loggingService, twoSecondLinearBackoff),
+      createFn: () => getBranches(httpClient, loggingService, twoSecondLinearBackoff),
       ttl: configCacheTtl,
     );
 
     return String.fromCharCodes(cacheValue).split(',');
   }
 
-  // Returns LUCI builders.
-  Future<List<LuciBuilder>> luciBuilders(String bucket, String repo,
-      {String commitSha = 'master', int prNumber}) async {
+  // Returns list of LUCI builders supported in Cocoon.
+  Future<List<LuciBuilder>> luciBuilders(String bucket, String repo, {String commitSha, int prNumber}) async {
     final GithubService githubService = await createGithubService('flutter', repo);
-    return await getLuciBuilders(githubService, Providers.freshHttpClient, twoSecondLinearBackoff, loggingService,
-        RepositorySlug('flutter', repo), bucket,
-        prNumber: prNumber, commitSha: commitSha);
+    final List<LuciBuilder> builders = <LuciBuilder>[];
```

Comment:
is this not going to break callers of this function by now returning all builders for all branches?

Reply:
No. `LuciService` calls this in [getBranchRecentTasks](https://cs.opensource.google/flutter/cocoon/+/master:app_dart/lib/src/service/luci.dart;l=67?q=luci&ss=flutter%2Fcocoon) to create `Map<BranchLuciBuilder, Map<String, List<LuciTask>>>`. However, because we branch builders as well, the branch keys contain nothing outside of `config.defaultBranch`

Reply:
With this change, the new scheduler seems to create new tasks for stable builders each time a master PR is merged.